concepticondata/concepticon.tsv

```diff
@@ -4057,4 +4057,37 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 4071	THEFT	Law	The unlawful taking of another’s property without consent and with the intent to permanently deprive them of it.	Person/Thing
 4072	DRAWER	The house	A sliding compartment in furniture, such as a desk or cabinet, used for storing items.	Person/Thing
 4073	WHEELBARROW	Agriculture and vegetation	A small hand-propelled vehicle with one wheel at the front and two handles at the back, used to transport soil, tools, or other heavy materials.	Person/Thing
-4074	DOLPHIN-LIKE WHALE (DOLPHIN OR PORPOISE)	Animals	Any of the related aquatic mammals within the superfamily Delphinoidea, such as dolphins or porpoises.	Person/Thing	
\ No newline at end of file
+4074	DOLPHIN-LIKE WHALE (DOLPHIN OR PORPOISE)	Animals	Any of the related aquatic mammals within the superfamily Delphinoidea, such as dolphins or porpoises.	Person/Thing
+4075	MOMENT	Time	A brief but measurable period of time, often considered important or significant in a particular context.	Person/Thing
+4076	DEVELOPMENT	Cognition	The process of growth, progress, or change in various domains, including technical, physical, emotional, cognitive, social, cultural, economic, or biological aspects.	Person/Thing
+4077	TRIANGLE	Basic actions and technology	A geometric shape with three straight sides and three angles, whose measures together sum to 180 degrees, often used in mathematics, geometry, and design.	Person/Thing
+4078	QUALITY	Cognition	A property or characteristic of a person, object, or substance that can be judged, measured, or perceived.	Person/Thing
+4079	LIGHTHOUSE	Modern world	A tall structure, typically on a coast, with a powerful light to guide ships and warn of hazards.	Person/Thing
```

Comment:
Lighthouses have been used for a long time... maybe `Basic actions and technology`? Either is fine though I'd say.